recipes/libtheora/meta.yaml

```diff
@@ -44,6 +44,7 @@ test:
 
 about:
   home: https://theora.org
+  license: BSD-like
```

Comment:
ok, will do this, but https://github.com/xiph/theora/blob/master/LICENSE is not really a BSD-3.